app/Services/SearchService.php

```diff
@@ -261,7 +261,13 @@ public function find()
             return $this->newOptions();
         }
 
-        return $searchResults;
+        foreach ($searchResults as $result) {
+            if (strtolower($result['name']) == $cleanTerm) {
+                return $searchResults;
+            }
+        }
+
+        return $searchResults + $this->newOptions();
```

Comment:
This now offers the "new" option in the entity filters for example (add an entity to an ability), which isn't exactly what we want, and shows only one result if I type in "goblin" when I have a race called "goblin" and "hobgoblin", which also isn't great :)